logstash-core/src/main/java/org/logstash/Cloner.java

```diff
@@ -1,6 +1,13 @@
 package org.logstash;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
```

Comment:
Is this a compiler warning, or a style change ?

Reply:
style change :) `.*` imports are just annoying